src/config-site/windows.cmake

```diff
@@ -205,7 +205,8 @@ VISIT_OPTION_DEFAULT(VISIT_H5PART_LIBDEP
 ##
 ## MFEM
 ##
-VISIT_OPTION_DEFAULT(VISIT_MFEM_DIR     ${VISITHOME}/mfem/3.3)
+VISIT_OPTION_DEFAULT(VISIT_MFEM_DIR     ${VISITHOME}/mfem/3.4)
+VISIT_OPTION_DEFAULT(VISIT_MFEM_INCDEP CONDUIT_INCLUDE_DIR TYPE STRING)
 
```

Comment:
actually one q: Where is CONDUIT_INCLUDE_DIR set? Is that prior to the option being set?